SWLOR.Game.Server/Feature/QuestDefinition/ViscaraQuestDefinition.cs

```diff
@@ -512,5 +513,33 @@ private void TaxiTerminalRepairs()
 
                 .AddKeyItemReward(KeyItemType.TaxiHailingDevice);
         }
+
+        private void JoiningTheRepublic()
+        {
+            _builder.Create("joining_the_republic", "Joining the Republic")
+
+                .AddState()
+                .AddKillObjective(NPCGroupType.Viscara_DeepMountainRaivors, 10)
+                .SetStateJournalText("Lieutenant Marbury Grant has instructed you to thin out the raivors that roam the Viscaran mountains to prove yourself worthy of enlisting in the Republic's Special Forces.")
+
+                .AddState()
+                .SetStateJournalText("You have hunted down ten raivors. Return to the Lieutenant to continue your path towards enlisting as a soldier of the Republic.")
+
+                .AddState()
+                .SetStateJournalText("Lieutenant Marbury Grant has instructed you to speak to a training droid located in the mess hall at Outpost Hope.")
+
+                .AddState()
+                .SetStateJournalText("You have completed the Lieutenant's test. Return to the Lieutenant to continue your path towards enlisting as a soldier of the Republic.")
+
+                .AddState()
+                .SetStateJournalText("Lieutenant Marbury Grant has instructed you to speak to Sergeant Nahulu, who awaits you are the parade square of Outpost Hope to reaffirm your oath of allegiance.")
+
+                .AddState()
+                .SetStateJournalText("You have reaffirmed your oath of allegiance to the Republic and the Senate. Return to the Lieutenant and conclude your enlistment as a soldier of the Republic.")
+
+                .AddXPReward(10000)
```

Comment:
Drop a zero on this XP reward. It's still insanity.

Reply:
Zunath, after your latest update to quest rewards, 10,000 is on the high side but not extraordinarily so.

War with the Mandalorian Warriors give 8,000.
The Mandalorian Leader gives 6,000.
Stuff Keeps Breaking gives 8,000.
Vanquish the Vellen Raiders give 12,000. 
Even Kath Hound hunting gives 3,000 and Help the Talyron Family gives 6,000.

I want it to be high because I want to test the idea of those joining factions and being bound to those factions having a leg-up at the start. This quest quite literally binds them to the Republic faction, with in-quest warnings of permanent death upon betrayal, etc. It's not a quest that every PC should or can do at all...